pint/timing_utils/statistics.py

```diff
@@ -0,0 +1,50 @@
+# Licensed under the  BSD 3-clause license - see LICENSE
+
+
+""" f_tests.py defines a set of functions for examing the timing model
+    parameters using F test.
+"""
+
+
+import numpy as np
+from scipy.special import fdtr
+
+
+def Ftest(chi2_1, dof_1, chi2_2, dof_2):
+    """Compute an F-test to see if a model with extra parameters is significant
+       compared to a simpler model.
+
+       Parameters
+       ----------
+       chi2_1: float
+           Original model's non-reduced chi^2.
+       dof_1: float
+           Original model's degree of freedom.
+       chi2_2: float
+           New model's non-reduced chi^2.
+       dof_2: float
+           New model's degree of freedom.
+
+       Return
+       ------
+       The probability that the improvement in chi2 is due to random chance
+       (i.e. a low probability means that the new fit is quantitatively better,
+       while a value near 1 means that the new model should likely be rejected).
+       If the new model has a higher chi^2 than the original model, returns
+       value is -1.
+
+       Note
```

Comment:
Should probably also mention that the models must be *nested*, that is, that model 1 must be obtainable from model 2 by freezing a certain number of the parameters of model 2. In fact it might make sense to replace one of the parameters dof_1, dof_2 by frozen_parameters to encourage users to be aware of this restriction.